src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsSynchronisationService.kt

```diff
@@ -85,50 +86,43 @@ class NonAssociationsSynchronisationService(
       return
     }
 
-    /* TODO investigate if need primary check
     if (event.isNotPrimaryNonAssociation()) {
       telemetryClient.trackEvent(
         "non-association-delete-synchronisation-non-primary-skipped",
         event.toTelemetryProperties(),
       )
       return
     }
-    */
 
-    telemetryClient.trackEvent(
-      "non-association-delete-synchronisation-success",
-      event.toTelemetryProperties(),
-    )
+    nonAssociationsMappingService.findNomisNonAssociationMapping(
+      firstOffenderNo = event.offenderIdDisplay,
+      secondOffenderNo = event.nsOffenderIdDisplay,
+      nomisTypeSequence = event.typeSeq,
+    )?.let {
+      log.debug("Found non-association mapping: $it")
+      log.debug("Sending non-association delete sync for " + it.nonAssociationId)
```

Comment:
would be better to either using string interpolation or use `{}` rather than adding strings together e.g.
```
      log.debug("Sending non-association delete sync for {}", it.nonAssociationId)
```